Packs/FeedRSS/ReleaseNotes/1_0_14.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### RSS Feed
+
+- Fixed an issue where the ***fetch-indicators*** command skip indicators creation without published value.
```

Comment:
```suggestion
Fixed an issue where the ***fetch-indicators*** command skipped indicators creation without published value.
```

Not sure I understand this.
The command did not fetch indicators that did not have a published value?

Reply:
yes, the indicators without the `published` value didnt fetched

Reply:
Then I would change this to:
Fixed an issue where indicators without a published value were not fetched by the ***fetch-indicators*** command.